.github/workflows/ci.yml

```diff
@@ -48,7 +48,13 @@ jobs:
           echo ttf-mscorefonts-installer msttcorefonts/accepted-mscorefonts-eula select true | sudo debconf-set-selections
           sudo apt-get install ttf-mscorefonts-installer
 
-      - name: Install package
```

Comment:
@andrzejnovak added the check to make sure it install properly without the `all`, at least on a minimal level